### DISCUSSION TITLE
Mac version error message pointing to 4.5.1 download link

### DISCUSSION THREAD

**Discussion by dstessel** [DistroAV/DistroAV#902] [General]
After fresh install of OBS and obs-ndi via BREW, I noticed that the error message for NDI Runtime Not Found is pointing to version 4.5.1 on  the developer's website. This is different from the [download location](libNDI_5.5.3_for_Mac.pkg) and file [described in the README](https://github.com/obs-ndi/obs-ndi/commit/08adaa5dc6d94e228d65ede0522b840d471e4521).

I saw that the official NDI redirect links were added in April.  It appears that this is not reflected on the BREW-downloaded versions. 

![image](https://github.com/obs-ndi/obs-ndi/assets/67884906/05ebeb03-be94-47eb-a41f-b29d502595b7)

For me, the version number was not displayed during the process. The file downloaded and installed, but versioning is indistinguishable (4.5.x vs 5.x).  This could explain some of the issues described by other users who may have assumed they were downloading the latest runtime version.

I am running the latest version of OBS on Mac. 
MacBook Pro Max
Ventura 13.5zb 

![image](https://github.com/obs-ndi/obs-ndi/assets/67884906/bf0f8a95-01c2-4ca7-8fb7-e4dd52698cc6)

Hope this is helpful, and thanks for all the work around obs-ndi! It's been incredibly useful.

**Comment by Trouffman** [comment]
Not sure who maintains the brew/cask/formular for the plugin.
This seems an old message from an old version.

**Comment by MarximusMaximus** [comment]
Attached is the mac installer for 4.5.1.

[ndi-runtime-4.5.1-macOS.pkg.zip](https://github.com/user-attachments/files/16277583/ndi-runtime-4.5.1-macOS.pkg.zip)
